runtime/README.md

```diff
@@ -711,8 +711,83 @@ Command Code: `0x434B_4558` ("CKEX")
+| preamble\_vendor\_lms\_key    | u32[6]       | Vendor LMS key                                                              |
+| preamble\_vendor\_ecc\_sig    | u32[24]      | Vendor LMS signature                                                        |
+| preamble\_vendor\_LMS\_sig    | u32[1344]    | Vendor LMS signature                                                        |
+| preamble\_vendor\_ecc\_key    | u32[24]      | Owner ECC key with R and S coordinates in that order                        |
+| preamble\_vendor\_lms\_key    | u32[6]       | Owner LMS key                                                               |
+| preamble\_vendor\_ecc\_sig    | u32[24]      | Owner LMS signature                                                         |
+| preamble\_vendor\_LMS\_sig    | u32[1344]    | Owner LMS signature                                                         |
+| metadata\_vendor\_ecc\_sig    | u32[24]      | Metadata Vendor LMS signature                                               |
+| metadata\_vendor\_LMS\_sig    | u32[1344]    | Metadata Vendor LMS signature                                               |
+| metadata\_owner\_ecc\_sig     | u32[24]      | Metadata Owner LMS signature                                                |
+| metadata\_owner\_LMS\_sig     | u32[1344]    | Metadata Owner LMS signature                                                |
+| metadata\_header\_revision    | u32          | Revision of the metadata header                                             |
+| metadata\_header\_reserved    | u32[3]       | Reserved                                                                    |
+| metadata\_entry\_entry\_count | u32          | number of metadata entries                                                  |
+| metadata\_entries             | MetaData[16] | The max number of metadata is 16 but less can be used                       |
+
+
+*Table: `AUTH\_MANIFEST\_FLAGS` input flags*
+
+| **Name**                  | **Value** |
+|---------------------------|-----------|
+| VENDOR_SIGNATURE_REQUIRED | 1 << 0    |
+
+*Table: `AUTH\_MANIFEST\_METADATA\_ENTRY` digest entries*
+
+| **Name**      | **Type** | **Description**        |
+|---------------|----------|------------------------|
+| digest        | u32[48]  | Digest of the metadata |
+| image\_source | u32      | Image source           |
+
+*Table: `SET\_AUTH\_MANIFEST` output arguments*
+
+| **Name**      | **Type** | **Description**
+| --------      | -------- | ---------------
+| chksum        | u32      | Checksum over other output arguments, computed by Caliptra. Little endian.
+| fips\_status  | u32      | Indicates if the command is FIPS approved or an error.
+
 
+### AUTHORIZE\_AND\_STASH
+
+Command Code: `0x4154_5348` ("ATSH")
+
+*Table: `AUTHORIZE\_AND\_STASH` input arguments*
+| **Name**    | **Type** | **Description**                                                             |
```

Comment:
The regular STASH_MEASUREMENT call has a `context` field:

|  |  | |
| -- | -- | --| 
| context | u8[48] | Context field for svn; e.g., a hash of the public key that authenticated the SVN. |

This is used in the stable identity proposal.

I think we should include that in this command as well.